modules/server/src/main/scala/observe/server/Systems.scala

```diff
@@ -358,7 +359,8 @@ object Systems {
 
     def build(site: Site, httpClient: Client[IO]): Resource[IO, Systems[IO]] =
       for {
-        webSocketBackend                           <- clue.http4sjdk.Http4sJDKWSBackend[IO]
+        clt                                        <- JdkWSClient.simple[IO]
```

Comment:
In other places we have used the `EmberClient` but I'm not sure if there is `WS` support. Is that the reason to use `JDKWSClient`?

Reply:
Oh no. It's just that when I asked Raúl what to use instead of `clue.http4sjdk.Http4sJDKWSBackend`, he pointed me to an example that used `JdkWSClient`.